README.md

```diff
@@ -1,39 +1,149 @@
+# 🧭 Bjorn Detector
 
-I have created an application that detects **Bjorn** on the local network, displays its IP address, and allows you to initiate an SSH session with a single click on Bjorn Icon.
+![Python](https://img.shields.io/badge/Python-3.9%2B-blue.svg)
+![Status](https://img.shields.io/badge/Status-Development-blue.svg)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+[![Reddit](https://img.shields.io/badge/Reddit-Bjorn__CyberViking-orange?style=for-the-badge&logo=reddit)](https://www.reddit.com/r/Bjorn_CyberViking)
+[![Discord](https://img.shields.io/badge/Discord-Join%20Us-7289DA?style=for-the-badge&logo=discord)](https://discord.com/invite/B3ZH9taVfT)
+
+an application that detects [Bjorn](https://github.com/infinition/bjorn) device on the local network, displays its IP address, and allows you to initiate an SSH session with a single click on Bjorn Icon.
 The red dot is moving around the radar when he looks for Bjorn.
 
-## Features
+## 📚 Table of Contents
+
+- [Features](#-features)
+- [Installation](#-installation)
+- [Usage](#-usage)
+  - [Initialization](#-initialization)
+- [Logging](#-logging)
+- [License](#-license)
+- [Contact](#-contact)
+
+## ✨ Features
 
 - **Automatic Network Detection**: Continuously checks for the device `bjorn.home` on the local network.
 - **Interactive SSH Launcher**: When `bjorn.home` is detected, click the Bjorn icon to automatically launch an SSH terminal connected to Bjorn’s IP address.
 - **IP Display**: Shows the IP address of `bjorn.home` upon detection.
 - **Seamless Bjorn Installation**: Facilitates  installation of [Bjorn](https://github.com/infinition/Bjorn/) by connecting to the device via SSH.
 
-## Requirements
+## 📌 Prerequisites
+
+- **Python**: 3.9+
+- **PyQt6**: 6.7.0+
+
+## 🛠️ Installation
+
+1. **Clone the Repository:**
+
+   ```bash
+   git clone https://github.com/infinition/bjorn-detector.git
+   cd bjorn-detector
+   ```
+
+2. **Create a Virtual Environment**
+
+   ```bash
+   python -m venv venv
+   ```
 
-- **Python 3**
-- **PyQt6**
+3. **Activate the Virtual Environment**
 
-## Installation
+   On Unix or MacOS:
 
-1. Clone this repository:
    ```bash
-   git clone https://github.com/infinition/Bjorn_Detector.git
-   cd Bjorn_Detector
+   source venv/bin/activate
    ```
 
-2. Install the required Python packages:
+   On Windows:
+
+   ```bash
+    .\venv\Scripts\activate
+   ```
+
+   - or
+
+   ```bash
+    powershell.exe -ExecutionPolicy Bypass -File .\venv\Scripts\Activate.ps1
+   ```
+
+4. **Upgrade pip**
+
+   ```bash
+   pip install --upgrade pip
+   ```
+
+5. **Install Dependencies**
+
    ```bash
    pip install -r requirements.txt
    ```
 
-## Launch
+   - Deactivate the Virtual Environment
+
+   When you're done, deactivate the environment:
+
+   ```bash
+    deactivate
+   ```
+   
+## 🚀 Usage
+
+### 📦 Initialization
+
+1. Prepare Input HTML Files:
```

Comment:
I think it's not related to Bjorn Detector, HTML translating thing ;)

Reply:
yes, I just maked related changes
[Commit Changes](https://github.com/infinition/bjorn-detector/pull/1/commits/781e0a7e83b0808e1d30468b2b6465af7ce234be)